es-core/src/components/BrightnessInfoComponent.h

```diff
@@ -17,7 +17,7 @@ class BrightnessInfoComponent : public GuiComponent
 	void render(const Transform4x4f& parentTrans) override;
 	void update(int deltaTime) override;
 
-	void reset() { mBrightness = 1; }
```

Comment:
Here's why this matters https://github.com/JustEnoughLinuxOS/emulationstation/blob/main/es-core/src/components/BrightnessInfoComponent.cpp#L97